tests/PHPExif/Mapper/ImageMagickMapperTest.php

```diff
@@ -605,5 +605,69 @@ public function testNormalizeComponentCorrectly()
           }
       }
 
+    /**
+     * @group mapper
+     * @covers \PHPExif\Mapper\ImageMagick::mapRawData
+     */
+    public function testMapRawDataCorrectlyKeywords()
+    {
+        $rawData = array(
+            \PHPExif\Mapper\ImageMagick::KEYWORDS => 'Keyword_1 Keyword_2',
+        );
+
+        $mapped = $this->mapper->mapRawData($rawData);
+
+        $this->assertEquals(
+            ['Keyword_1 Keyword_2'],
+            reset($mapped)
+        );
+    }
+
+    /**
+     * @group mapper
+     * @covers \PHPExif\Mapper\ImageMagick::mapRawData
+     */
+    public function testMapRawDataCorrectlyKeywordsAndSubject()
+    {
+        $rawData = array(
+            \PHPExif\Mapper\ImageMagick::KEYWORDS => array('Keyword_1', 'Keyword_2'),
+        );
+
+        $mapped = $this->mapper->mapRawData($rawData);
+
+        $this->assertEquals(
+            array('Keyword_1' ,'Keyword_2'),
+            reset($mapped)
+        );
+    }
 
+    /**
+     * @group mapper
+     * @covers \PHPExif\Mapper\ImageMagick::mapRawData
+     */
+    public function testMapRawDataCorrectlyFormatsXResolution()
+    {
+        $rawData = array(
+            \PHPExif\Mapper\ImageMagick::XRESOLUTION => '1500/300',
```

Comment:
I have already been wondering above why we need to scan for a slash in this value. I am obviously no export in EXIF tags. Intuitively, I assumed that the value is a primitive integer in DPI. Obviously, it is not. Just out of curiosity: What does a value '1500/300' mean?

Reply:
Again, I copied this one from Native: https://github.com/LycheeOrg/php-exif/blob/2bd2d3482cca296fc477fe1a07d1f465e145264c/tests/PHPExif/Mapper/NativeMapperTest.php#L245-L254

I have honestly no idea why it does things this way. In fact, when I was copying the code in Mapper from Native to ImageMagick, I replaced all this scanning for a slash nonsense with a call to `normalizeComponent`: https://github.com/LycheeOrg/php-exif/blob/2bd2d3482cca296fc477fe1a07d1f465e145264c/lib/PHPExif/Mapper/Native.php#L366-L377

But then the above test failed, because `5.0 != 1500`. And it's pretty clear that the test deliberately checks for this, so I decided not to mess with it, even though it would've been nice to have a comment in there explaining why things are done the way they are...